frameworks/non-keyed/seed/src/lib.rs

```diff
@@ -119,37 +120,41 @@ fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg>) {
             model.rows.clear();
             model.rows.reserve(count);
             for _ in take_units(count) {
-                model.rows.push(Row::new(&mut model.previous_id, &mut model.generator));
+                model
+                    .rows
+                    .push(Row::new(&mut model.previous_id, &mut model.generator));
             }
-        },
+        }
         Msg::Append(count) => {
             model.rows.reserve(count);
             for _ in take_units(count) {
-                model.rows.push(Row::new(&mut model.previous_id, &mut model.generator));
+                model
+                    .rows
+                    .push(Row::new(&mut model.previous_id, &mut model.generator));
             }
-        },
+        }
         Msg::Update(step) => {
             for position in (0..model.rows.len()).step_by(step) {
                 model.rows.get_mut(position).unwrap().label += " !!!";
             }
-        },
+        }
         Msg::Clear => {
             model.rows.clear();
-        },
+        }
         Msg::Swap => {
             if model.rows.len() > 998 {
                 model.rows.swap(1, 998);
             }
-        },
+        }
         Msg::Select(id) => model.selected = Some(id),
         Msg::Remove(id) => {
             model.rows.remove(Row::position(id, &model.rows));
-        },
+        }
     }
     orders.force_render_now();
```

Comment:
@MartinKavik Could you explain why it is required here to force the rendering?

Reply:
If I remember correctly it was a way to skip waiting for the browser render loop (aka `requestAnimationFrame`). Yew doesn't / didn't use it at all so I was thinking it makes the results more comparable.
But I'm not sure if we need / want it. Feel free to remove it.